docs/content/developer/getting-started/connect.mdx

```diff
@@ -163,7 +169,7 @@ test your software in real-world conditions.
 
```

Comment:
Move `<DataWipe />` to Devnet

Reply:
Ohh, good catch. Fixed in 3c02fcdb2af574de07480bf868489f183623912c